project/plugins.sbt

```diff
@@ -3,10 +3,8 @@ resolvers += Resolver.typesafeRepo("releases")
 resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.3.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.9.0")
 
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
-
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7" exclude("org.slf4j", "slf4j-simple"))
-
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.16" exclude("org.slf4j", "slf4j-simple"))
```

Comment:
do we still need the slfj exclusion?

Reply:
Unclear. But we always have it in all repositories, and I'm too scared to remove it.